classes/publication/maps/Schema.php

```diff
@@ -96,7 +96,9 @@ function ($publicationFormat) use ($submissionFiles, $genres) {
             );
         }
 
-        $output = $this->schemaService->addMissingMultilingualValues(PKPSchemaService::SCHEMA_PUBLICATION, $output, $this->context->getSupportedSubmissionLocales());
+        $locales = $publication->getLanguages($this->context->getSupportedSubmissionMetadataLocales());
```

Comment:
Shouldn't here be as in OJS:
`$allowedLocales = $submission->getpublicationLanguages($context->getSupportedSubmissionMetadataLocales());`

Maybe it would be good to comment some things:
I believe here are locales from all existing publications important -- same as in publication Repository::add, while in the publication Repository::edit only that publication locales are important?